addons/common/CfgFunctions.hpp

```diff
@@ -141,6 +141,12 @@ class CfgFunctions
                 description = "Drops a weapon.";
                 file = "\x\cba\addons\common\fnc_dropWeapon.sqf";
             };
+            // CBA_fnc_execNextFrame
+            class execNextFrame
+            {
```

Comment:
Opening brace on same line as class. I know it's not used in others in this file but it has to get converted anyways.